frontend/packages/console-dynamic-plugin-sdk/src/api/api.ts

```diff
@@ -1,4 +1,5 @@
 import { UseK8sWatchResource, UseK8sWatchResources } from './api-types';
```

Comment:
Can we also `git mv` `src/api/api-types.ts` :arrow_right: `src/api/types/index.ts` ?

Reply:
Can be done later.